src/components/Article/Article.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BoxProps } from '../Box';
+import Section from '../Section';
+
+/**
+ * Increases the heading level of all `H` components inside it by 1.
+ *
+ * NOTE: this component is functionally the same as <Section />, but semantically
+ * it renders an <article> tag instead of a <section> tag.
+ */
+export const Article: React.FC<BoxProps<'article'>> = ({ as = 'article', ...rest }) => {
```

Comment:
I imagine these need their own example `mdx` files too? Will be essentially the same as the `H` example

Reply:
Yeah. every `components/{COMPONENT}` folder needs:

- a test file
- an mdx file for docs (even if minimal)